FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,20 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		self.client.get(from: url) { [weak self] clientResult in
+
+			guard self != nil else { return }
+
+			switch clientResult {
+			case .success((let data, let response)):
```

Comment:
This success `case` can be simplified with a single let.
```suggestion
			case let .success((data, response)):
```